.github/workflows/ci-workflow.yml

```diff
@@ -12,39 +12,25 @@ jobs:
     permissions:
       pull-requests: read
     outputs:
-      javascript_changed: ${{ steps.filter.output.javascript }}
+      javascript_changed: ${{ steps.filter.outputs.javascript }}
     steps:
       - uses: actions/checkout@v4
       - uses: dorny/paths-filter@v3
         id: filter
         with:
           filters: |
             javascript:
-              - '**/*.js'
```

Comment:
verify nice

Reply:
it just occurred to me that if we only update a dep this CI won't run. but that would also cause the prep release PR to run because the package.json will change as well so maybe we need both?